ArchiSteamFarm/Bot.cs

```diff
@@ -150,6 +150,10 @@ public sealed class Bot : IDisposable {
 		private string TwoFactorCode;
 		private byte TwoFactorCodeFailures;
 
+		internal bool HasWallet { get; private set; }
+		internal int Balance { get; private set; }
+		internal string Currency { get; private set; }
```

Comment:
Use enum here, call `ToString()` when needed instead, 4 bytes is cheaper than 4 bytes per character. Also rename to `WalletCurrency`.